docs/en/thrift.md

```diff
@@ -0,0 +1,140 @@
+[中文版](../cn/thrift.md)
+
+[thrift](https://thrift.apache.org/)is a RPC framework used widely in production area which was developed by Facebook, In order to access thrift servers more conveniently and make full use of bthread's capability of concurrency, brpc directly supports the thrift protocol.

```

Comment:
production area ->  production environment
Facebook, -> Facebook.
bthread's capability of concurrency -> concurrency ability of bthread

Reply:
Thanks, Updated.